src/main/resources/templates/admin.html

```diff
@@ -10,7 +10,7 @@
     </style>
 </head>
 
-<body>
+<body class="d-flex flex-column min-vh-100">
```

Comment:
This is also in most of the files, where you can designate a flexbox container on the body that allows the footer-spacer to work properly.